>>> 'Pkg.add("Revealables")' log
INFO: Cloning cache of Interact from git://github.com/JuliaLang/Interact.jl.git
INFO: Cloning cache of Revealables from git://github.com/sisl/Revealables.jl.git
INFO: Installing DataStructures v0.3.11
INFO: Installing Docile v0.5.15
INFO: Installing FactCheck v0.3.1
INFO: Installing Interact v0.1.6
INFO: Installing Reactive v0.2.2
INFO: Installing Revealables v0.0.1
INFO: Building Revealables
=============================[ ERROR: Revealables ]=============================

LoadError: IPython 3 is required for Revealables.
could not spawn `ipython --version`: no such file or directory (ENOENT)
while loading /home/vagrant/.julia/v0.4/Revealables/deps/build.jl, in expression starting on line 7

================================================================================

================================[ BUILD ERRORS ]================================

WARNING: Revealables had build errors.

 - packages with build errors remain installed in /home/vagrant/.julia/v0.4
 - build the package(s) and all dependencies with `Pkg.build("Revealables")`
 - build a single package by running its `deps/build.jl` script

================================================================================
INFO: Package database updated

>>> 'Pkg.test("Revealables")' log
Julia Version 0.4.0-dev+6661
Commit 9c3cb36 (2015-08-12 03:21 UTC)
Platform Info:
  System: Linux (x86_64-unknown-linux-gnu)
  CPU: Intel(R) Core(TM) i7-4960HQ CPU @ 2.60GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas
  LIBM: libopenlibm
  LLVM: libLLVM-3.3
INFO: Testing Revealables
ERROR: LoadError: ArgumentError: NBShowable not found in path
 in require at ./loading.jl:193
 in include at ./boot.jl:254
 in include_from_node1 at ./loading.jl:264
 in process_options at ./client.jl:308
 in _start at ./client.jl:411
while loading /home/vagrant/.julia/v0.4/Revealables/test/runtests.jl, in expression starting on line 1
=============================[ ERROR: Revealables ]=============================

failed process: Process(`/home/vagrant/julia/bin/julia --check-bounds=yes --code-coverage=none --color=no /home/vagrant/.julia/v0.4/Revealables/test/runtests.jl`, ProcessExited(1)) [1]

================================================================================
ERROR: Revealables had test errors
 in error at ./error.jl:21
 in test at pkg/entry.jl:746
 in anonymous at pkg/dir.jl:31
 in cd at file.jl:22
 in cd at pkg/dir.jl:31
 in test at pkg.jl:71
 in process_options at ./client.jl:284
 in _start at ./client.jl:411

>>> End of log
